frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageComments.vue

```diff
@@ -39,7 +39,7 @@
         </v-card-text>
         <v-card-actions class="justify-end mt-0 pt-0">
           <v-btn
-            v-if="user.id == comment.user.id || user.admin"
+            v-if="user.id == comment.user.id || (user.admin && user.groupId === recipe.groupId)"
```

Comment:
Do we want user.admin to be its own true condition? Regardless of the group. (Like previously)

I don't have a strong opinion.

Reply:
Yeah after thinking it over changing it back makes sense. I was thinking more of a group-level admin vs a site-level admin, but we don't distinguish between the two